descriptors/ModuleDescriptor-template.json

```diff
@@ -29,7 +29,7 @@
             "source-storage.records.put",
             "specification-storage.specifications.collection.get",
             "specification-storage.specification.rules.collection.get",
-            "ui-quick-marc.settings.lccn-duplicate-check.edit",
+            "mod-settings.global.read.ui-quick-marc.lccn-duplicate-check.manage",
```

Comment:
@AndreiBordak I don't think "edit" permission is needed for duplicate check. "read" permission should be enough.